### ISSUE TITLE
Move while drinking toggle

### ISSUE THREAD

**Issue by yaboifuso** [mikewii/Luma3DS-plugin#18]
Allow toggling of walking while drinking (Potions/armor skins/demon drugs/antidote etc. like monster hunter world. This should be limited to walking only, not sprinting as well and this CANT be cancelled early like in MHW, the cancel early possibly being it’s own toggle for those who want the exact mhw experience

**Comment by MisterPrintf** [comment]
I don't think that this could be implemented while online; it could potentially crash other players.

**Comment by yaboifuso** [comment]
That’s why it’s a toggle, there when you want it - gone when you don’t.
